h/static/scripts/login-forms/routes.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * URL patterns for routes that the frontend router needs to match or
+ * dynamically generate links for.
+ *
+ * These must be synchronized with h/routes.py.
+ */
+export const routes = {
+  login: '/login',
+  forgotPassword: '/forgot-password',
+  signup: '/signup',
+};
```

Comment:
We should combine this and `group-forms/routes.ts` in future since the routes have to be unique across all apps.